lib/datadog/ci/ext/environment.rb

```diff
@@ -213,16 +213,19 @@ def extract_github_actions(env)
           ref = env['GITHUB_REF'] if ref.nil? || ref.empty?
```

Comment:
We're modifying `extract_github_actions`, but `extract_travis` and `extract_appveyor` also hardcode github.com. Would it make sense to expand the `env['GITHUB_SERVER_URL']` for those?

Or are they only usable with github.com, not the enterprise solution, and thus that wouldn't apply there?

Reply:
good catch. They're indeed only usable in github (both github.com and enterprise solution). `GITHUB_SERVER_URL` wouldn't be present as an environment variable in travis and appveyor